fili-core/src/test/groovy/com/yahoo/bard/webservice/table/PhysicalTableSpec.groovy

```diff
@@ -150,4 +150,14 @@ class PhysicalTableSpec extends Specification {
         expect:
         physicalTable.getDimensions() == [dimension] as Set
     }
+
+    def "test physical to logical mapping is constructed correctly"() {
+        setup:
+        def oneDimPhysicalTable = new PhysicalTable("test table", DAY.buildZonedTimeGrain(UTC), ['dimension':'druidDim'])
+        def twoDimPhysicalTable = new PhysicalTable("test table", DAY.buildZonedTimeGrain(UTC), ['dimension1':'druidDim', 'dimension2':'druidDim'])
```

Comment:
Unless we have good reason to be taking advantage of Groovy's duck-typing, we should be explicitly specifying the types of the variables.